src/obis-observer/commands/uplink/GetObserverCapabilitiesResponse.ts

```diff
@@ -0,0 +1,113 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import CommandBinaryBuffer, {REQUEST_ID_SIZE, ICommandParameters} from '../../CommandBinaryBuffer.js';
+import {UPLINK} from '../../constants/directions.js';
+
+
+/**
+ * IGetObserverCapabilitiesResponseParameters command parameters
+ */
+interface IGetObserverCapabilitiesResponseParameters extends ICommandParameters {
+    maxMeterProfilesNumber: number,
+    maxMetersNumber: number,
+    maxObisProfilesNumber: number,
+    multiModeSupported: boolean
```

Comment:
```suggestion
    isMultiModeSupported: boolean
```

?

the current variant is also acceptable

Reply:
did it